gui/src/main/java/org/verapdf/gui/utils/ValidateWorkerSummary.java

```diff
@@ -0,0 +1,102 @@
+package org.verapdf.gui.utils;
+
+import org.verapdf.component.AuditDuration;
+import org.verapdf.processor.reports.BatchSummary;
+import org.verapdf.processor.reports.FeaturesBatchSummary;
+import org.verapdf.processor.reports.MetadataRepairBatchSummary;
+import org.verapdf.processor.reports.ValidationBatchSummary;
+
+
+/**
+ * @author Artem Kiriutin
+ */
+public class ValidateWorkerSummary implements BatchSummary {
```

Comment:
make this summary an inner class of Validate worker and do not implement Batch summary